controls/radchartview/axes/categoricalaxis.md

```diff
@@ -42,8 +65,56 @@ The categorical axis displays a range of categories and the values that determin
 	      </telerik:LineSeries.DataPoints>
 	   </telerik:LineSeries>
 	</telerik:RadCartesianChart.Series>
-	{{endregion}}
+{{endregion}}
+
+Notice how each data point is its own category on the X axis.

```

Comment:
I think we won't need this sentence if we change the description above the snippet.